.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+
+# Performance optimization
+git:
+  depth: 10
+cache:
+  directories:
+    - $HOME/.npm
```

Comment:
In my experience, this does not really improve performance, since the cached directories have to be restored to/saved from the CI container which tends to take even longer than just downloading them in the first place.